rgw/admin/radosgw.go

```diff
@@ -28,19 +28,24 @@ var (
 	errNoSecretKey = errors.New("secret key not set")
 )
 
+// HTTPClient interface that conform to the http package
+type HTTPClient interface {
+	Do(req *http.Request) (*http.Response, error)
+}
+
 type verbHTTP string
```

Comment:
Not related to this PR, but this can be removed since https://github.com/ceph/go-ceph/pull/531/ got merged right?